app/components/search/thumbnail_component.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if show_thumbnail? %>
+  <%= image_tag thumbnail_url, class: 'thumbnail', alt: '' %>
```

Comment:
Note: The alt here must be an empty string in order to notify screen readers that this is a decorative element for accessibility, if `alt: nil` is used, the `alt` will not be included in the rendered element.